### DISCUSSION TITLE
Rename variables/methods without modifying them directly

### DISCUSSION THREAD

**Discussion by HundSimon** [dnSpyEx/dnSpy#327] [Q&A] [answered]
Hello! I’m new to using dnSpy, and I was wondering if it’s possible to rename variables and methods in dnSpy similar to how it’s done in IDA Pro. In IDA Pro, renaming doesn’t actually change the variable or method names; it’s more like adding a comment. This feature is particularly useful when analyzing obfuscated programs.

**Comment by MarkVanAcker** [comment, marked as answer]
In case you quickly need a solution or this, this plugin implements what you asked for: https://github.com/holly-hacker/dnSpy.Extension.HoLLy
(It works or version 6.3, but unsure about 6.5)
